docs/features.md

```diff
@@ -0,0 +1,311 @@
+
+Default settings:
+
+```bash
+"CUSTOM_DOMAINS": {
+  "isEnabled": true,
+  "selectors": [
+    {
+      "type": "apiGroup",
+      "apiGroup": "dns.gardener.cloud"
+    }
+  ]
+},
+```
+
+- **EVENTING** – is used to show or hide the Eventing view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"EVENTING ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"eventing.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **KUBECONFIG_ID** – is used to configure URL to which Busola sends a request to download a kubeconfig file. If we add `?kubeconfigID={your ID}` to the Busola URL then Busola tries to download the kubeconfig from the {kubeconfigUrl}/{yourID} and, if succedded, will add a cluster.
+  You can introduce the whole address in the kubeconfigUrl which will also be read by Busola.
+
+Default settings:
+
+```bash
+"KUBECONFIG_ID": {
+ "isEnabled": true,
+ "config": {
+   "kubeconfigUrl": "https://kyma-env-broker.cp.dev.kyma.cloud.sap/kubeconfig"
+ }
+},
+```
+
+- **LEGAL_LINKS** – is used to show or hide legal links. You can find all available links in the below example.  
+  In config you can find the unchangeable keys (you cannot use legalDisclosure instead of legal-disclosure). The keys include the default link, which takes you to the default address, and a link that depends on your chosen language.
+
+Example:
+
+```bash
+"LEGAL_LINKS": {
+  "config": {
+    "legal-disclosure": {
+      "default": "https://www.sap.com/corporate/en/legal/impressum.html",
+      "de": "https://www.sap.com/corporate/de/legal/impressum.html"
+    },
+    "privacy": {
+      "default": "https://help.sap.com/viewer/82bdf2271c6041f79387c122147cf774/Cloud/en-US"
+    },
+    "copyright": {
+      "default": "https://www.sap.com/corporate/en/legal/copyright.html",
+      "de": "https://www.sap.com/corporate/de/legal/copyright.html"
+    },
+      "trademark": {
+        "default": "https://www.sap.com/corporate/en/legal/trademark.html",
+        "de": "https://www.sap.com/corporate/de/legal/trademark.html"
+    }
+  }
+},
+```
+
+A link under the given key will be selected based on your language code (de, en, pl, etc.), If the code is not available, the default link is used.
+
+- **OBSERVABILITY** – is used to render a few nodes in the navigation. The label shows the name of the given service. The path is used by Busola during the bootstrapping. Busola sends a request to the cluster address + the path and the cluster must return the virtualService object. If the object is found you receive an address to which the node in the navigation leads.
```

Comment:
Do we want to specify that the address you receive is a host from the virtual service object?

Should it be `VirtualService`?

Reply:
I'd leave that technical detail here, as it's not likely to change.

Of course, you're right with the upper-casing.